testing/zerotier-one/APKBUILD

```diff
@@ -0,0 +1,67 @@
+# Contributor: Kyle Parisi <kyleparisi@gmail.com>
+# Maintainer: Kyle Parisi <kyleparisi@gmail.com>
+pkgname=zerotier-one
+pkgver=1.1.14
+pkgrel=0
+pkgdesc="ZeroTier One allows systems to join and participate in ZeroTier virtual networks."
+url="https://www.zerotier.com/"
+arch="all"
+license="GPL3"
+depends=""
+depends_doc="nodejs bash"
```

Comment:
Do not use depends_doc here (read below)
Do you really need bash to build this, looks like you can patch shebang in doc/build.sh to /bin/sh (maybe send this upstream)